OpenNefia.Content/CharaMake/CharaMakeLayer.cs

```diff
@@ -0,0 +1,145 @@
+﻿using OpenNefia.Content.UI;
+using OpenNefia.Content.UI.Element;
+using OpenNefia.Content.UI.Element.Containers;
+using OpenNefia.Content.UI.Element.List;
+using OpenNefia.Core.Audio;
+using OpenNefia.Core.Locale;
+using OpenNefia.Core.Maths;
+using OpenNefia.Core.Rendering;
+using OpenNefia.Core.UI;
+using OpenNefia.Core.UI.Element;
+using OpenNefia.Core.UI.Layer;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.CharaMake
+{
+    public class CharaMakeLayer : UiLayerWithResult<CharaMakeData, CharaMakeResult>, ICharaMakeLayer
+    {
+        protected class AttributeIcon : UiElement
+        {
+            private readonly Dictionary<string, string> _attributes = new Dictionary<string, string>
+            {
+                { "Elona.StatStrength", "0" },
+                { "Elona.StatConstitution", "1" },
+                { "Elona.StatDexterity", "2" },
+                { "Elona.StatPerception", "3" },
+                { "Elona.StatLearning", "4" },
+                { "Elona.StatWill", "5" },
+                { "Elona.StatMagic", "6" },
+                { "Elona.StatCharisma", "7" }
+
+            };
+            private IAssetInstance AssetAttributeIcons;
+            private string Type;
+
+            public AttributeIcon(string type)
+            {
+                AssetAttributeIcons = Assets.Get(AssetPrototypeOf.AttributeIcons);
+                Type = type;
+            }
+
+            public override void Draw()
+            {
+                base.Draw();
+                GraphicsEx.SetColor(Color.White);
+                if (_attributes.TryGetValue(Type, out var iconId))
+                    AssetAttributeIcons.DrawRegion($"{iconId ?? "2"}", X, Y, centered: true);
```

Comment:
Kinda want this `"2"` to be a named constant, even if it's just for this layer class.

Reply:
done (76f6762)